editor/inspector_dock.cpp

```diff
@@ -399,6 +399,23 @@ void InspectorDock::_resource_selected(const Ref<Resource> &p_res, const String
 	EditorNode::get_singleton()->push_item(r.operator->(), p_property);
 }
 
+void InspectorDock::_files_moved(const String &p_old_file, const String &p_new_file) {
+	// Because only the file name is shown, we care about changes on the file name.
+	if (p_old_file.get_file() != p_new_file.get_file()) {
```

Comment:
Pretty sure this is always true.

Reply:
If you move a file from one directory to another without changing the name, that condition will be false. Because the inspector only shows the name, we can ignore those moves.

Reply:
Ah, I missed `get_file()`.
You can change it to early return though.